standard-app/templates/workloads/cronjob.yaml

```diff
@@ -116,13 +117,10 @@ spec:
                 {{- end }}
               {{- end }}
               env:
-                {{- range $key, $value := $container.env }}
+                {{- range $key, $value := merge (default dict $cronjobConfig.env) (default dict $.Values.env) }}
```

Comment:
let's also add `$container.env` to this merge - they can be specified at container level too